.github/workflows/main.yml

```diff
@@ -0,0 +1,15 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  update_pr:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: deepakputhraya/action-pr-title@master
```

Comment:
I wonder if there is a way we can somehow control these actions without forking and managing them ourselves. Maybe pin the specific version to start with